LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -221,21 +237,35 @@ public void CreateClassrooms(int classroomCount)
         {
             if (ModuleIdList.Count != 0)
```

Comment:
Add an **else** block to report the error: missing modules.

Reply:
Done